### PR TITLE
Remove Contacts and make account names unchangeable

### PR DESCRIPTION
**Pull request by davidyuk**

### CODE REVIEW

**Comment by etharner** [comment]
image
Looks like new default name longer than Main Account and it leads to breaking text to new line

**Comment by etharner** [comment]
image
It seems account name missed here

File: src/components/mobile/AeAccountReverse.vue [hunk 1]
```diff
@@ -34,8 +35,12 @@ export default {
   },
   props: {
     address: { type: String, required: true },
-    name: { type: String, default: '' },
+    source: { type: Object, required: true },
```
Comment (etharner) [inline src/components/mobile/AeAccountReverse.vue]: `sourse` is not used in this component

**Comment by etharner** [comment]
image
After switching to other branch names are missed

File: src/components/mobile/AeAccountReverse.vue (hunk 1 above)
Comment (davidyuk) [reply src/components/mobile/AeAccountReverse.vue]: Actually, it is needed to calculate `getName(this)`

**Comment by davidyuk** [comment]
> After switching to other branch names are missed, reset helps but maybe need migration 

Migration is not needed because account names will be removed automatically on the first sync with local storage.

**Pull request merged**